### PR TITLE
US 1583733 Add missing language IDs - 63

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for sample program output displayed on the console.

Contributes to #2192

### CODE REVIEW

**Comment by v-thpra** [comment]
@mairaw I added the language ID's to code blocks that did not have them. These files were part of the CATS report.

File: spec/conversions.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ End Module
 
 The output of the program is:
 
-```
+```console
```
Comment (hartmair) [inline spec/conversions.md]: As far as i know there is no Markdown language `console`. Did you mean `bash`? See https://highlightjs.org/static/demo/

Comment (v-thpra) [reply spec/conversions.md]: That's right, it is not a language. However, "console" is on the list of [supported language IDs](https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#supported-languages). In order to put a descriptive ID on these code blocks and avoid a P0 CATS report error, we have been using "console" for examples of output that are displayed to the console (like in this case), and for examples of command line tools.

Comment (hartmair) [reply spec/conversions.md]: Alright, agree, better than nothing.

**Comment by v-thpra** [comment]
@hartmair @BillWagner It looks like these changes are accepted and reviewed. It would be great to get this merged. Thanks!

**Pull request merged**